conf/puppet.conf

```diff
@@ -0,0 +1,9 @@
+[main]
+    # Where Puppet stores dynamic and growing data.
+    vardir=/opt/puppetlabs/agent/cache
+
+    # The Puppet log directory.
+    logdir=/var/log/puppetlabs/agent
+
+    # Where Puppet PID files are kept.
+    rundir=/var/run/puppetlabs
```

Comment:
You may want to create one of these specifically for Windows (unless it is already there).